### PR TITLE
add pymkl

### PR DESCRIPTION
**Pull request by ocefpaf**
Adding `pymkl`. Ping @lheagy.

### CODE REVIEW

**Comment by ocefpaf** [comment]
@conda-forge/staged-recipes this is ready for review.

File: recipes/pymkl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+package:
+  name: pymkl
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/pyMKL/pyMKL-{{ version }}.tar.gz
+  sha256: 5a24f63f510007d908506c88aa5b6b720a0d9201b8222d14805aca1f771b6afa
+
+build:
+  number: 0
```
Comment (isuruf) [inline recipes/pymkl/meta.yaml]: `noarch: python` ?

Comment (ocefpaf) [reply recipes/pymkl/meta.yaml]: I think it can be. Let's try.

**Comment by ocefpaf** [comment]
@isuruf are you familiar with this package? This good to go?

**Comment by ocefpaf** [comment]
Thanks @isuruf!

**Pull request merged**